<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifica</class>
 <widget class="QDialog" name="DialogModifica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>474</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="nome_lay">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="nome_text">
         <property name="maximumSize">
          <size>
           <width>9999</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="nome_lay_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Cognome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="cognome_text">
         <property name="maximumSize">
          <size>
           <width>999999</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="maximumSize">
          <size>
           <width>99999</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="nome_lay_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QRadioButton" name="uomo_radio">
         <property name="text">
          <string>Uomo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="donna_radio">
         <property name="maximumSize">
          <size>
           <width>437</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Donna</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="nome_lay_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="occupazione_label">
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Occupazione</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="occupazione_text">
         <property name="maximumSize">
          <size>
           <width>9999</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>456</width>
       <height>24</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogModifica</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogModifica</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
